client/modules/datafiles/src/publications/PublicationSearchToolbar/PublicationSearchToolbar.tsx

```diff
@@ -40,58 +86,71 @@ export const PublicationSearchToolbar: React.FC = () => {
-          <i className="fa fa-search" />
-          &nbsp;Search
+          <i className="fa fa-search" /> &nbsp;Search
         </Button>
         <Form.Item name="query" style={{ marginBottom: 0, flex: 1 }}>
           <Input
             placeholder="Author, Title, Keyword, Description, Natural Hazard Event, or Project ID"
             style={{ flex: 1 }}
+            onFocus={() => setShowTips(true)}
+            autoComplete="off"
           />
         </Form.Item>
       </Form>
-      <div>
-        <label htmlFor="nh-type-select" style={{ margin: 0 }}>
-          Natural Hazard Type
-        </label>
-        &nbsp;
-        <Select
-          style={{ width: '150px' }}
-          virtual={false}
-          allowClear
-          id="nh-type-select"
-          placeholder="All Types"
-          options={[
-            { label: 'All Types', value: null },
-            ...dropdownOptions.nhTypeOptions,
-          ]}
-          popupMatchSelectWidth={false}
-          value={searchParams.get('nh-type')}
-          onChange={(v) => setSearchParam('nh-type', v)}
-        />
-      </div>
-
-      <div>
-        <label htmlFor="publication-year-select" style={{ margin: 0 }}>
-          Year Published
-        </label>
-        &nbsp;
-        <Select
-          style={{ width: '150px' }}
-          id="publication-year-select"
-          options={[{ label: 'All Years', value: null }, ...yearOptions]}
-          allowClear
-          placeholder="All Years"
-          popupMatchSelectWidth={false}
-          virtual={false}
-          value={searchParams.get('pub-year')}
-          onChange={(v) => setSearchParam('pub-year', v)}
-        />
-      </div>
 
-      <Button type="link" onClick={() => setSearchParams(undefined)}>
-        Clear Filters
-      </Button>
+      {showTips && (
+        <Collapse
+          bordered={false}
+          defaultActiveKey={['1', '2']}
+          style={{ width: '100%' }}
+        >
+          <Panel header="Search Tips" key="1">
+            <p>
+              <strong>"exact phrase"</strong> - Use quotes to search for exact
+              phrases.
+            </p>
+            <p>
+              <strong>word1 OR word2</strong> - Find results containing either
+              word.
+            </p>
+            <p>
+              <strong>word1 AND word2, word3 word4</strong> - Use AND, commas,
+              and spaces to find results containing each word.
+            </p>
+          </Panel>
+          <Panel header="Recent Searches" key="2">
+            {recentSearches.length ? (
+              recentSearches.map((term, index) => (
+                <div
+                  key={index}
+                  style={{
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                  }}
+                >
+                  <p
+                    style={{ cursor: 'pointer', color: 'blue', margin: 0 }}
```

Comment:
The CSS default `blue` is darker than the links we use elsewhere in the Data Depot. There should be a CSS variable for the color we want to be using.